MoltenVK/MoltenVK/Utility/MVKEnvironment.cpp

```diff
@@ -44,17 +44,26 @@ static constexpr uint32_t getExpectedMVKConfigurationStringCount() {
 
 #pragma mark Set configuration values
 
+// Expand the shortcut versions, Add the VK_HEADER_VERSION.
+static uint32_t expandAPIVersion(uint32_t apiVer) {
+	switch (apiVer) {
+		case  0:  return VK_API_VERSION_1_0;
+		case 10:  return VK_API_VERSION_1_0;
+		case 11:  return VK_API_VERSION_1_1;
+		case 12:  return VK_API_VERSION_1_2;
+		case 13:  return VK_API_VERSION_1_3;
+		default:  return std::min(apiVer, VK_API_VERSION_1_3);
```

Comment:
Shouldn't this be `MVK_VULKAN_API_VERSION` instead of hardcoding 1.3, as it was before? Also, these shorthands feel a little weird to me tbh...

Reply:
Fixed. Thanks.